### ISSUE TITLE
UV-mapping merged or multiple tiff files on a mesh

### ISSUE THREAD

**Issue by kristalamaria** [domlysz/BlenderGIS#208]
Hi, a lot of thanks for this amazing plug in. I've been trying to develop a 3d topographic map and encountered a problem when importing a texture in my mesh. I have broken down my topomap into several georeferenced tiff files in qgis (just like a tile puzzle) to have the resolution that I want. I initially planned on importing the tiff files one by one, but when I imported the first one, my whole mesh was automatically filled just like in the figure below.
![blender-gis](https://user-images.githubusercontent.com/57040352/67621713-23e2aa80-f845-11e9-910a-0d6f98c4722c.png)
Afterwards, I tried to merge the georeferenced tiff files on qgis so that I can just simply import a single file, but it turned out that the only image overlayed in the mesh is the very first tiff file on the list just like below.
![blender-gis2](https://user-images.githubusercontent.com/57040352/67621833-3a3d3600-f846-11e9-9f26-24fb77695f30.png)
I have attempted a few more times, tried to import the merged tiff file on a plane and I can't view the merged tiff file at all. I have also read from the pdf guidelines that one 1 blender unit = coordinate system unit, so I thought that this might have something to do with the unit of my recently imported file. I tried to play around with the sequence of my import and with the 'Adjust 3d view' feature on Blender Preferences, but I can't seem to get out of the loop. If this issue is something that has been raised before, I ask for forgiveness. Thank you very much.

**Comment by domlysz** [comment]
Hi, you need to switch the 3d view shading mode to "look dev" or "render preview" to correctly see the uv mapping
![Sans titre](https://user-images.githubusercontent.com/5639024/67936993-0ad35400-fbcd-11e9-91b2-fd2e0a0afebe.jpg)

**Comment by kristalamaria** [comment]
Hi domlysz, thank you for responding. So I have tried your advice and been successful in displaying a single georef image in its rightful place,
![blender-gis3](https://user-images.githubusercontent.com/57040352/67946468-65e06780-fc1c-11e9-8758-6a37205a3554.png)
but it seems like when I import more georef tiles, they are not showing up in the mesh. Just that one georef tile that I first imported :/ Thank you for your patience, I am relatively new to blender.

**Comment by domlysz** [comment]
Yes it's because at each import it create a new material and assign all mesh's faces to it. So, for now the addon is not abble to deal with this use case.

A quick workaround can be through shader nodes, but with lot of tiles it will be a pain

![image](https://user-images.githubusercontent.com/5639024/67950042-36fcce00-fbe9-11e9-8856-2c02bf7d90c7.png)

If you really need tiling meshes here some clues : https://github.com/domlysz/BlenderGIS/issues/172
but it's not easy to achieve, what's the problem with the non-tiled topomap ???

**Comment by kristalamaria** [comment]
Hi domlysz, thank you for referencing the tiled terrain project. I will surely take a look into it. The details of the topomap is just too massive to fit in a single tile. Perhaps I will just consider adjusting the resolution of my project if I can't make this work. Thank you very much for your hard work.

**Comment by domlysz** [comment]
TODO do not assign basemap material to the first indexed material
